README.md

```diff
@@ -98,12 +104,16 @@ $ kubectl describe vm testvm
 
 # To start a VM you can use, this will create a VM instance (VMI)
 $ ./virtctl start testvm
+# OR with krew/virt installed
+$ kubectl virt start testvm
```

Comment:
Let's not disturb the general reading flow by having both otions for each operation.

Instead, please add a paragraph above noting that if krew is used, then all appearances of `virtctl` should be replaced with `kubectl virt`

- fabian

Reply:
@fabiand feedback addressed, please rereview!